tests/expected/assert-arg-error/test.rs

```diff
@@ -1,10 +0,0 @@
-// Copyright Kani Contributors
```

Comment:
Can you turn this into fixme test instead?

Reply:
We currently don't have a way to add fixme tests that are passing, but should actually fail (see discussion on https://github.com/model-checking/kani/pull/1666). That's why I deleted it. I'm not happy about that though, so I'm open to ideas on how we can keep the test in our regressions in a meaningful way.

Reply:
Honestly, I just rename the test, add a comment and change the expected file to the current output. It won't be part of a fixme suite, but I think it's better than deleting it.